### PR TITLE
clean up settings page

### PR DESCRIPTION
**Pull request by HParker**
I did a couple things here.

* made it look a little different:
screen shot 2015-03-18 at 9 39 49 pm

* live update while changing the site name
* xhr's only happen once when clicking save
* new save animation:
screen shot 2015-03-18 at 9 39 13 pm
* name in header now links to homepage

### CODE REVIEW

**Comment by HParker** [comment]
Also didn't run the tests...

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -11,6 +11,6 @@
 // GO AFTER THE REQUIRES BELOW.
 //
 //= require jquery
-//= require jquery_ujs
+// require jquery_ujs
```
Comment (HParker) [inline app/assets/javascripts/application.js]: For some reason including jquery_ujs causes xhrs to happen twice.

File: app/views/settings/index.html.erb [hunk 2]
```diff
@@ -16,23 +22,37 @@
 
 <div class="row">
   <div class="col-md-6">
```
Comment (HParker) [inline app/views/settings/index.html.erb]: Sorry this bit of the diff is messy. I just moved some things around to make the page more even.

**Comment by HParker** [comment]
I'll look into fixing the build later. the tests that are still broken are passing locally. except for the music one, which seems unrelated and more like a broken test.

File: spec/features/settings_page_spec.rb [hunk 3]
```diff
@@ -1,96 +1,97 @@
+# -*- coding: utf-8 -*-
 require "rails_helper"
 
 feature "settings page" do
-	scenario "user changes the site name", js: true do
```
Comment (XanderStrike) [inline spec/features/settings_page_spec.rb]: :space_invader:

**Comment by XanderStrike** [comment]
:+1: 

Looks awesome. Feel free to merge when the build is green. I don't know what's up with those errors, they just upgraded selenium and Firefox on the build machines so that might have something to do with it. I love the new button.

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (XanderStrike) [inline app/assets/javascripts/application.js]: Weird. Was it doing that before and I just never noticed?

Comment (XanderStrike) [inline app/assets/javascripts/application.js]: Actually I just pulled it to play with it and your xhr doesn't work at all unless I reenable this. Also, with it enabled it doesn't seem to be submitting them twice.

Comment (HParker) [inline app/assets/javascripts/application.js]: Weird. I think how it is working for you is correct. I just guessed at this solution from http://stackoverflow.com/questions/6659209/rails-js-controller-being-called-twice-for-some-reason

I wonder if my jQuery is setup strangely?

File: spec/features/settings_page_spec.rb (hunk 3 above)
Comment (HParker) [inline spec/features/settings_page_spec.rb]: haha killin' tabs!

File: spec/features/settings_page_spec.rb [hunk 4]
```diff
@@ -1,96 +1,97 @@
...
-	scenario "user changes music dir setting", js: true do
-		page = SettingsPage.new
-		page.set_music_dir '/srv/music'
-		expect(page).to have_content('Saved')
-		expect(Setting.get(:music_dir).content).to eq('/srv/music')
-	end
+  scenario "user changes the site name", js: true do
+    page = SettingsPage.new
+    page.set_name 'Testtop Theater'
+    expect(has_button?('Save ✓')).to equal(true)
```
Comment (HParker) [inline spec/features/settings_page_spec.rb]: This is the only real change here.
`expect(page).to have_content('Saved')` -> expect(has_button?('Save ✓')).to equal(true)

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (HParker) [inline app/assets/javascripts/application.js]: Either way, I'll put this back and just make this change locally until I figure out what is going on with my system.

**Comment by HParker** [comment]
My local dev setup seems to have gone through some flux. I think we are good here now. Merging.

**Pull request merged**